types.d.ts

```diff
@@ -64,6 +64,19 @@ export type EmojiMatchThemes = {
   [key in EmojiMatchThemeKey]: EmojiMatchThemeValue;
 };
 
+export type DailyPuzzles = {
```

Comment:
How do I test the typing again? Not sure I defined all these properly.

Reply:
VS Code should do it inline automatically, but I'll add a script. We have a few issues

Reply:
Cool. I loaded it in VS and fixed some of the type problems